cypress/unit/store-mutation-selectCategoricalOption.cy.js

```diff
@@ -24,10 +29,9 @@ describe("selectCategoricalOption mutation", () => {
     it("Makes sure a value map is created for a column in the data dictionary", () => {
 
         // Act
-        mutations.selectCategoricalOption(store.state, {
-            optionValue: "https://example.org/female",
-            columnName: "column1",
-            rawValue: "F"
+        mutations.alterColumnCategoryMapping(store.state, {
```

Comment:
Since this case now uses `alterColumnCategoryMapping`, I think it's better to move it to `store-mutation-alterColumnCategoryMapping.cy.js` unit-test. What do you think?